client/scanner.js

```diff
@@ -63,6 +73,17 @@ class Scanner {
     return true
   }
 
+  async isExecutable(txRequest) {
```

Comment:
We can use the txRequest.inExecutionWindow() check here from eac.js-lib.

Reply:
update implemented:
https://github.com/adibas03/eac.js-client/blob/a418a8e772cc5af2c601ff92ef21f2a215cb4666/client/scanner.js#L73